webapp/webpack.common.js

```diff
@@ -92,10 +92,9 @@ function makeCommonConfig() {
                 publicPath: '{{.BaseURL}}/',
             }),
         ],
-        entry: {
-            main: './src/main.tsx',
-        },
+        entry: ['./src/main.tsx', './src/userSettings.ts'],
         output: {
+            library: "Focalboard",
```

Comment:
Minor / Lint: Should use single quotes

Reply:
Of course. Not sure how I missed all the single quotes around this line. 🤦